[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_light_push_with_no_timestamp with id: 2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d
[35mDEBUG   [0m src.steps.common:common.py:17 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.light_push:light_push.py:27 Running fixture setup: light_push_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:18 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node1_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:21 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:24 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:81 Generated random external IP 172.18.243.232
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:74 Generated ports ['25686', '25687', '25688', '25689', '25690']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:48 docker run -i -t -p 25686:25686 -p 25687:25687 -p 25688:25688 -p 25689:25689 -p 25690:25690 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=25688 --rest-port=25686 --tcp-port=25687 --discv5-udp-port=25689 --rest-address=0.0.0.0 --nat=extip:172.18.243.232 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=afcdae92ec74f0ba79ddfe5ddcddab0b5548aebeef2201b40a7cb3dad822da70 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=25690 --metrics-logging=true --lightpush=true --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:54 docker network connect --ip 172.18.243.232 waku e4fc08230cd41dad0d0c99d198f8598dad192b5f34cb7afcfb0e564d242a54f6
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:57 Container started with ID e4fc08230cd4. Setting up logs at ./log/docker/receiving_node1_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 25686
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:25686/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:25686/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.243.232/tcp/25687/p2p/16Uiu2HAm2iPipfivG4iMD7zhELKnvn6ibyCnZpyt8YAZnioRcxjB","/ip4/172.18.243.232/tcp/25688/ws/p2p/16Uiu2HAm2iPipfivG4iMD7zhELKnvn6ibyCnZpyt8YAZnioRcxjB"],"enrUri":"enr:-LO4QEDgc1w6z-5XdYI7FLG4TeSzLJT6c5hV58B40xWAh8LTWREjlATR7_ga2CNTXrWFbs3GlFbXJVH8TuhQetY6qRUBgmlkgnY0gmlwhKwS8-iKbXVsdGlhZGRyc4wACgSsEvPoBmRY3QOCcnOFAAMBAACJc2VjcDI1NmsxoQJsSkyThyCdvHYle39wsUu5QhaznkSOSnG2hbnDf0xSwoN0Y3CCZFeDdWRwgmRZhXdha3UyCQ"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:18 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node2_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:21 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:24 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:81 Generated random external IP 172.18.109.40
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:74 Generated ports ['47390', '47391', '47392', '47393', '47394']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:48 docker run -i -t -p 47390:47390 -p 47391:47391 -p 47392:47392 -p 47393:47393 -p 47394:47394 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=47392 --rest-port=47390 --tcp-port=47391 --discv5-udp-port=47393 --rest-address=0.0.0.0 --nat=extip:172.18.109.40 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=63c6bf38dbdb6bedc136daa0ef03e4b3ffbcab579c2fda105f2a39a0b95dd19a --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=47394 --metrics-logging=true --lightpush=false --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:54 docker network connect --ip 172.18.109.40 waku 1fd2fbb61f12acde529284def349415e2593cd2d269ea648d4f3af5016f27fff
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:57 Container started with ID 1fd2fbb61f12. Setting up logs at ./log/docker/receiving_node2_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 47390
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:47390/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:47390/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.109.40/tcp/47391/p2p/16Uiu2HAmDRqGDYkKExwBqtmsdSWm4GiKRhrHNqNx5hgjxA5MRhNZ","/ip4/172.18.109.40/tcp/47392/ws/p2p/16Uiu2HAmDRqGDYkKExwBqtmsdSWm4GiKRhrHNqNx5hgjxA5MRhNZ"],"enrUri":"enr:-LO4QBhU0RxP0aeGv88vPwakwmTGxDKhX4-ACGdDg7tHnYFxAiCMHhGqvWoM0_GBoSWlm0CQ19VI2SUn1IbIypZ0mRkBgmlkgnY0gmlwhKwSbSiKbXVsdGlhZGRyc4wACgSsEm0oBrkg3QOCcnOFAAMBAACJc2VjcDI1NmsxoQMLfNx2iP6mX1yv0TdglrO28vuUBV36Rfa6jP-khugrwoN0Y3CCuR-DdWRwgrkhhXdha3UyAQ"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:47390/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.243.232/tcp/25687/p2p/16Uiu2HAm2iPipfivG4iMD7zhELKnvn6ibyCnZpyt8YAZnioRcxjB"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:18 Docker client initialized with image wakuorg/go-waku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/lightpush_node1_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:21 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:24 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:81 Generated random external IP 172.18.210.10
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:74 Generated ports ['42441', '42442', '42443', '42444', '42445']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:48 docker run -i -t -p 42441:42441 -p 42442:42442 -p 42443:42443 -p 42444:42444 -p 42445:42445 wakuorg/go-waku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=DEBUG --rest-relay-cache-capacity=100 --websocket-port=42443 --rest-port=42441 --tcp-port=42442 --discv5-udp-port=42444 --rest-address=0.0.0.0 --nat=extip:172.18.210.10 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=38ad704fa7aeded634b5ec4a0d3bf6bd86de4a4fcf68adce9e15dd0ec73f40bc --min-relay-peers-to-publish=1 --rest-filter-cache-capacity=50 --peer-store-capacity=10 --discv5-bootstrap-node=enr:-LO4QEDgc1w6z-5XdYI7FLG4TeSzLJT6c5hV58B40xWAh8LTWREjlATR7_ga2CNTXrWFbs3GlFbXJVH8TuhQetY6qRUBgmlkgnY0gmlwhKwS8-iKbXVsdGlhZGRyc4wACgSsEvPoBmRY3QOCcnOFAAMBAACJc2VjcDI1NmsxoQJsSkyThyCdvHYle39wsUu5QhaznkSOSnG2hbnDf0xSwoN0Y3CCZFeDdWRwgmRZhXdha3UyCQ --lightpushnode=/ip4/172.18.243.232/tcp/25687/p2p/16Uiu2HAm2iPipfivG4iMD7zhELKnvn6ibyCnZpyt8YAZnioRcxjB --lightpush=true --relay=false
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:54 docker network connect --ip 172.18.210.10 waku 141249d44230b5be176b9df3355ce703c6f156780f80fdfe3d31edcbf2653206
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:57 Container started with ID 141249d44230. Setting up logs at ./log/docker/lightpush_node1_2024-10-09_03-23-47__b1ba793e-d765-455e-b9d2-a86658d53f6d__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/go-waku:latest. REST: 42441
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:42441/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"enrUri":"enr:-Ji4QDRn5Y8vOos88OJiwtNGjSDFvRFGGSlsvY1YB6rzC1ruThYNytahPedvpoCzv_gdzc3vO6cQ3myVJzoGZz25eeqGAZJvTfLNgmlkgnY0gmlwhKwRAAqJc2VjcDI1NmsxoQIQPi0joLjQc_TjaKgweKbA1gGCnt2iL1ZzM0y4DYxgfIN0Y3CCpcqDdWRwgqXMhXdha3UyCA","listenAddresses":["/ip4/127.0.0.1/tcp/42442/p2p/16Uiu2HAkvX5VW68VyrVodP6b3uPukXnCJbXVc1Zp7Ks22VmdAGtj","/ip4/127.0.0.1/tcp/42443/ws/p2p/16Uiu2HAkvX5VW68VyrVodP6b3uPukXnCJbXVc1Zp7Ks22VmdAGtj","/ip4/172.17.0.10/tcp/42442/p2p/16Uiu2HAkvX5VW68VyrVodP6b3uPukXnCJbXVc1Zp7Ks22VmdAGtj","/ip4/172.17.0.10/tcp/42443/ws/p2p/16Uiu2HAkvX5VW68VyrVodP6b3uPukXnCJbXVc1Zp7Ks22VmdAGtj"]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:42441/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.243.232/tcp/25687/p2p/16Uiu2HAm2iPipfivG4iMD7zhELKnvn6ibyCnZpyt8YAZnioRcxjB", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:42441/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.109.40/tcp/47391/p2p/16Uiu2HAmDRqGDYkKExwBqtmsdSWm4GiKRhrHNqNx5hgjxA5MRhNZ", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:25686/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:47390/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.light_push:light_push.py:120 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:42441/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/3/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto"}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'true'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_1:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:25686/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":0,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_2:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:47390/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":0,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id e4fc08230cd4
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id 1fd2fbb61f12
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id 141249d44230
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:117 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:117 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:117 No errors found in the waku logs.